TYPO3.Flow/Classes/TYPO3/Flow/Utility/SchemaValidator.php

```diff
@@ -571,6 +571,21 @@ protected function validateAnyType($value, array $schema)
     }
 
     /**
+     * Create a string information for the given value
+     *
+     * @param $value
+     * @return string
+     */
+    protected function renderValue($value)
+    {
+        if (is_scalar($value)) {
+            return (string)$value;
+        } else {
+            return 'type=' . gettype($value);
+        }
```

Comment:
could be a one liner
`return is_scalar($value) ? (string)$value : 'type=' . gettype($value);`